example/lint.sh

```diff
@@ -22,9 +22,26 @@ fix=""
 buildevents=$(mktemp)
 filter='.namedSetOfFiles | values | .files[] | select(.name | endswith($ext)) | ((.pathPrefix | join("/")) + "/" + .name)'
 
+unameOut="$(uname -s)"
+case "${unameOut}" in
+    Linux*)     machine=Linux;;
+    Darwin*)    machine=Mac;;
+    CYGWIN*)    machine=Windows;;
+    MINGW*)     machine=Windows;;
+    MSYS_NT*)   machine=Windows;;
+    *)          machine="UNKNOWN:${unameOut}"
+esac
+
+args=()
+if [ $machine == "Windows" ]; then
+    # avoid missing linters on windows platform
```

Comment:
nit: you should run `pre-commit install` so this gets indented for you

Reply:
Didn't know about pre-commit hooks, I've learned something new, thank you! The buildifier hooks are failing on windows with missing /bin/bash errors. And all files are getting converted from CRLF to LF, I assume this is not desired?